.github/workflows/go-build.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build_linux:
+    # TODO: see if we can cross compile for other archs with CGo
+    runs-on: ubuntu-latest
```

Comment:
I always have trouble deciding between -latest and pinning a version. Both have downsides, but given you're just using a few github-provided actions, -latest is likely fine.

Reply:
That was my reasoning, indeed. If I would actually install extra packages, I would have pinned it.

Reply:
Decided to pin after all.